config/samples/monitoring_v1alpha1_clusterurlmonitor.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: monitoring.openshift.io/v1alpha1
+kind: ClusterUrlMonitor
+metadata:
+  name: clusterurlmonitor-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
maybe modify this file to express an actual ClusterUrlMonitor

Reply:
didn't know I needed to do this by hand